interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/use-report-parameters/date-range-parameters.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Date Range Parameters'
+author: Sergey Andreev
+---
+# Date Range Parameters
+
+This document describes how to create a date range parameter and filter report data by the specified dates.
+
+![](../../../../../images/eurd-win-use-date-ranges-parameters-panel.png)
+
+Perform the steps below to add a date range parameter to a report.
+
+1. Switch to the [Field List](../../report-designer-tools/ui-panels/field-list.md) and right-click **Parameters**. Select **Add Parameter** from the context menu.
+
+    ![](../../../../../images/eurd-win-use-date-ranges-design-add-param.png)
+
+1. Specify the options below in the invoked **Add New Parameter** dialog and click **OK**.
+
+    - **Name** - the parameter's name;
+    - **Description** - the parameter's description displayed in Print Preview's **Parameters** panel;
+    - **Type** - the parameter's data type. Set it to **Date** to create a date range;
+    - **Range Value** - enable this option to create a date range.
+
+    ![](../../../../../images/eurd-win-use-date-ranges-design-add-param-dialog.png)
+
+1. The _dateRange_ parameter appears in the **Field List** and includes the inner _dateRange_Start_ and _dateRange_End_ parameters. Select the _dateRange_ parameter and switch to the **Property Grid** to specify the inner parameters' default values.
+
+    ![](../../../../../images/eurd-win-use-date-ranges-design-configure-subparam.png)
+
+    The **Value** property allows you to specify a static default value. Provide an expression for the **Value** property to specify a dynamic default value. Click the **Value** property's marker, select **Value Expression** and specify an expression in the invoked editor.
```

Comment:
```suggestion
    The **Value** property allows you to specify a static value. To do this. provide an expression for the **Value** property which specifies a dynamic value. Click the **Value** property's marker, select **Value Expression**, and specify an expression in the invoked editor.
```

Reply:
Changed to:
    The **Value** property allows you to specify a static value. To specify a dynamic value, click the **Value** property's marker, select **Value Expression** and specify an expression in the invoked editor.